model/font.go

```diff
@@ -422,14 +422,28 @@ func (font *PdfFont) BytesToCharcodes(data []byte) []textencoding.CharCode {
 
 // CharcodesToUnicodeWithStats is identical to CharcodesToUnicode except returns more statistical
 // information about hits and misses from the reverse mapping process.
+// NOTE: The number of runes returned may be greater than the number of charcodes.
+// TODO(peterwilliams97): Deprecate?
```

Comment:
Maybe add a note to deprecate in a future major version (v4) and point to what alternative function to use, presumably CharcodesToRuneSlices

Reply:
Done